README.md

```diff
@@ -1,43 +1,89 @@
-# Cisco Spark JavaScript SDKs
+# spark-js-sdk
 
-## Ciscospark
+> Monorepo containing the Cisco Spark JavaScript SDKs (both current and legacy).
 
 [ciscospark](/packages/ciscospark) is a collection of node modules targeting our [external APIs](https://developers.ciscospark.com). Its core libraries take inspiration from our web client's Legacy SDK.
 
-```bash
-npm install --save ciscospark
-```
+This README primarily discusses the tooling required to develop the Cisco Spark SDK.
+
+## Table of Contents
 
-### Building
+- [Install](#Install)
+- [Usage](#Usage)
+- [Contribute](#Contribute)
+- [License](#License)
 
-At this time, a prebuilt version of ciscospark is not available. If your project already uses [browserify](http://browserify.org), or [webpack](https://webpack.github.io/) this shouldn't be an issue. If not, you'll need to clone the repository and build it with:
+## Install
+
+Install tooling dependencies with
 
 ```bash
-npm install -g browserify
 npm install
+```
+
+Install module dependencies with
+```bash
 npm run bootstrap
-npm run build
-browserify --standalone ciscospark packages/ciscospark > bundle.js
+```
+(This installs dependencies to packages/\*/node_modules)
+
+## Usage
+
+The following commands should be run from the SDK root directory. Tooling dependencies are typically defined only in the root node_modules directory to avoid duplicates. (eslint is installed in each package directory because many editors, or at least Atom, get confused by the monorepo layout and which eslint binary to use).
+
+### Build
+
+```bash
+PACKAGE=PACKAGENAME npm run grunt:package -- build
+```
+
+### Run tests
+
+```bash
+PACKAGE=PACKAGENAME npm run grunt:package -- test
 ```
 
-> For production builds, change `npm run build` to `NODE_ENV=production npm run build`.
+### Run just unit tests
+Handy during early plugin development when you can write a bunch of unit tests.
 
-Note that when when building for the browser, you'll need define
--  `CISCOSPARK_ACCESS_TOKEN`
-or
-- `CISCOSPARK_CLIENT_ID`
-- `CISCOSPARK_CLIENT_SECRET`
-- `CISCOSPARK_REDIRECT_URI`
-- `CISCOSPARK_SCOPE`
+```bash
+UNIT_ONLY=true PACKAGE=PACKAGENAME npm run grunt:package -- test
+```
+
+### Run the tests, but only in nodejs
+Usually faster, and can build on the fly, thus no need to rebuild everything between test runs
 
-Alternatively, you can use use `ciscospark.init()` to set those values at runtime. See Environment Setup (below) and the (ciscospark)[packages/ciscospark/README.md] README for more info on those values.
+```bash
+PACKAGE=PACKAGENAME npm run grunt:package -- express:test test:node
```

Comment:
The chat-widget doesn't have express tests, should we mention anything about jest tests here?